_posts/2022-04-07-launching-open-source-operator-for-openSearch.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Introducing open source Kubernetes Operator for OpenSearch"
+authors:
+  - Opster
+date: 2022-04-07
+categories:
+  - technical-post
+twittercard:
+  description: "The Kubernetes OpenSearch Operator is used for automating the deployment, provisioning, management, and orchestration of OpenSearch clusters and OpenSearch dashboards."
+---
+
+
+### Overview
+
+Since the launch of OpenSearch project, a fully open source k8s Operator has been a [popular request](https://discuss.opendistrocommunity.dev/t/kubernetes-operator-support-for-the-fork/5267)  with the community. We are happy to announce the [beta release](https://github.com/Opster/opensearch-k8s-operator/releases/tag/v0.9) for the OpenSearch Operator.
+
+The [OpenSearch Operator](https://github.com/Opster/opensearch-k8s-operator) is fully open-source kubernetes operator, licensed as Apache 2.0, and is used for automating the deployment, provisioning, management, and orchestration of OpenSearch clusters and OpenSearch dashboards.
+
+The Operator development is being led by [Opster](https://opster.com/) with partners including [SUSE Rancher](https://www.suse.com/), [Maibornwolff](https://www.maibornwolff.de/en), [AWS](https://aws.amazon.com/), [Logz.io](https://logz.io/) and more.
+
+### Operator Capabilities
+
+The Operator enables high-level API use for easily running advanced OpenSearch operations on Kubernetes.
+
+With this beta release, the Operator allows for management of multiple OpenSearch clusters and OpenSearch Dashboards. Using Operator makes scaling up and down, version upgrades, rolling restarts, adjustment of memory and disk resources on the nodes, securing deployments and managing certificates simplified and streamlined.
+
+In the future releases, it would also allow for advanced shard allocation strategies, monitoring with Prometheus and Grafana, control shard balancing and allocation (For example based on AZ/Rack awareness, Hot/Warm) and auto-scaling based on usage load and resources.
+
+### Getting Started
+The Operator is available at https://github.com/Opster/opensearch-k8s-operator.
+
+#### Step 1: Installation
+In order to install the Operator you can use 2 methods:
+    1. Helm chart and the other is to build it locally.
+    2. Local installation
+
+###### Helm chart installation
+- Download the helm gz file from https://github.com/Opster/opensearch-k8s-operator/releases/download/v0.9/opensearchOperator-chart.tar.gz
+- Follow these instructions: https://github.com/Opster/opensearch-k8s-operator#installing-the-operator-on-your-k8s-cluster-with-helm
```

Comment:
Should this be an active URL or do you intend the user to copy the URL from the post? Please update if it should be an active URL.